cunumeric/linalg/linalg.py

```diff
@@ -20,9 +20,8 @@
 from cunumeric._ufunc.math import add, sqrt as _sqrt
 from cunumeric.array import add_boilerplate, convert_to_cunumeric_ndarray
 from cunumeric.module import dot, empty_like, eye, matmul, ndarray
-from numpy.core.multiarray import (  # type: ignore [attr-defined]
-    normalize_axis_index,
-)
+from numpy.core.multiarray import normalize_axis_index  # type: ignore
+from numpy.core.numeric import normalize_axis_tuple  # type: ignore
```

Comment:
so evidently all imports from numpy inside `linalg.py` evince this problem with mypy. Still have not been able to determine why.